app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -215,13 +223,11 @@ class HomeMenu(
 
         // Only query account manager if it has been initialized.
         // We don't want to cause its initialization just for this check.
-        val accountAuthItem =
-            if (context.components.backgroundServices.accountManagerAvailableQueue.isReady() &&
-                context.components.backgroundServices.accountManager.accountNeedsReauth()) {
-                    reconnectToSyncItem
-            } else {
-                null
-            }
+        val accountAuthItem = if (context.components.backgroundServices.accountManagerAvailableQueue.isReady()) {
+            if (context.components.backgroundServices.accountManager.accountNeedsReauth()) reconnectToSyncItem else null
```

Comment:
Just for awareness I think this extra if statement would be the equivalent as the above using &&. This probably didn't need to be change.